core/trino-main/src/main/java/io/trino/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -792,7 +791,7 @@ private void verifyNoOrderByReferencesToOutputColumns(Node node, StandardErrorCo
         getReferencesToScope(node, analysis, orderByScope.get())
                 .findFirst()
                 .ifPresent(expression -> {
-                    throw semanticException(errorCode, expression, errorString);
+                    throw semanticException(errorCode, expression, "%s", errorString);
```

Comment:
same

Reply:
In this case I annotated the enclosing method as a `@FormatMethod` instead. It's more generic this way ;)